examples/hypercompute_clusters/a3u-gke-gcs/a3u-gke-gcs.yaml

```diff
@@ -0,0 +1,296 @@
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+blueprint_name: a3u-gke-gcs
+
+vars:
+  project_id: # Insert GCP project
+  deployment_name: # Unique name of this cluster, like a3u-gke-gcs
+  region: # Region, e.g. europe-west1
+  zone: # Zone, e.g. europe-west1-b
+
+  # Cidr block containing the IP of the machine calling terraform and kubectl
+  # The value can be more specific if the IPs are known which will run kubectl
+  # e.g. the local system running Terraform or a remote node
+  authorized_cidr: 0.0.0.0/0
+  extended_reservation:  # Reservation name, e.g. <project>/<reservation-name>/reservationBlocks/<reservation-block-name>
+
+  nccl_installer_path: $(ghpc_stage("./nccl-rdma-installer.yaml"))
+  mtu_size: 8896
+  static_node_count: # Number of A3-Ultra nodes, e.g. 2
+  # Number of H200 GPUs (for later use by Kueue), which
+  # should be 8 x `static_node_count`
+  num_gpus:
+  training_bucket_name: # Name of bucket that holds training data
+  checkpoint_bucket_name: # Name of bucket used for checkpoints
+  system_node_pool_disk_size_gb: 200
+  a3ultra_node_pool_disk_size_gb: 100
```

Comment:
100G seems too low for AI/ML images, this could lead to disk out of space issue and pod stuck in pulling image

Reply:
we seen this kind of issue with 200G space before, and end up using 500